public/static/docs/command-reference/gc.md

```diff
@@ -33,15 +33,15 @@ restored using `dvc fetch`, as long as they have previously been uploaded with
 
 ## Options
 
-- `-a`, `--all-branches` - keep cached objects referenced from the latest commit
-  across all Git branches. It should be used if you want to keep data for the
-  latest experiment revisions. Especially, if you intend to use `dvc gc -c` this
-  option is much safer.
+- `-a`, `--all-branches` - keep cached objects referenced from the latest
+  [revisions](https://git-scm.com/docs/revisions) across all Git branches (tip
+  commits). It should be used if you want to keep data for the latest experiment
+  revisions. Especially, if you intend to use `dvc gc -c` this option is much
```

Comment:
I would prefer even versions over revisions here. Too many fancy terms people are not aware and do not care about.

Reply:
Agree. This was a mistaken use of my criteria for "version" vs. "revision" actually, thanks for noticing. Reverted.